community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java

```diff
@@ -202,8 +242,6 @@ public void shouldGetCorrectDijkstraPathsWithWeights() throws Exception
         assertEquals( 1.5, secondPath.get( "weight" ) );
         // 5 = 3 +2
         assertEquals( 5, (Integer) firstPath.get( "length" ) + (Integer) secondPath.get( "length" ) );
-        assertThatPathHasLength( firstPath, 3 );
-        assertThatPathHasLength( secondPath, 2 );
```

Comment:
What about adding: ```assertEquals( 1, Math.abs( (Integer) firstPath.get( "length" ) - (Integer) secondPath.get( "length" ) ) );``` and a comment explaining that the assertions try not to rely on the order of results?